packages/lib-sourcify/src/lib/logger.ts

```diff
@@ -0,0 +1,58 @@
+export interface ILogger {
+  logLevel: number;
+  log: (level: number, message: string) => void;
+  setLevel: (level: number) => void;
+}
+
+// Default logger behavior
+export const DefaultLogger: ILogger = {
+  logLevel: 2,
```

Comment:
We should log info level, no?
```suggestion
  logLevel: 3,
```

Reply:
I don't like library that by default they log info, I see most of them simply log warn and err